### PR TITLE
[SERVER-1172] Removing references to Local Observabiluty Stack

### PR DESCRIPTION
**Pull request by soulchips**
# Description
Local Observability Stack has been reduced to only prometheus for now
Updating server 3.x docs to reflect this. Removing references to other
parts of the stack:
- loki
- grafana
- promtail

# Reasons
https://circleci.atlassian.net/browse/SERVER-1172

### CODE REVIEW

File: jekyll/_cci2/server-3-overview.adoc [hunk 1]
```diff
@@ -136,7 +136,7 @@ CircleCI Server 3.0 consists of the following services. Find their descriptions
 | circleci-telegraf
 |
 | Telegraf collects statsd metrics. All white-boxed metrics in our services publish statsd metrics that are sent to telegraf,
-but can also be configured to be exported to other places (i.e. Datadog or local-observability-stack-prometheus)
+but can also be configured to be exported to other places (i.e. Datadog or prometheus)
```
Comment (lloydChris) [inline jekyll/_cci2/server-3-overview.adoc:139]:
```suggestion
but can also be configured to be exported to other places (i.e. Datadog or Prometheus)
```

**Review by lloydChris — CHANGES_REQUESTED** [review]

**Review by lloydChris — CHANGES_REQUESTED** [review]
the Grafana and port-forwarding references need to be removed as well: https://github.com/circleci/circleci-docs/pull/5393/files#diff-6352b6ef7e33b2bb178451e6fb4ba4f3c8eabe6f9eb8c33da01e6f7753203efeR47

File: jekyll/_cci2/server-3-operator-metrics-and-monitoring.adoc [hunk 2]
```diff
@@ -27,22 +27,9 @@ cases. Hence, features around local observability are provided on an as-is basis
 flexibility and extensibility of the system, should you feel that the functionality provided is not sufficient for your
```
Comment (lloydChris) [inline jekyll/_cci2/server-3-operator-metrics-and-monitoring.adoc:27]: maybe a @nathanfishcircleci questions, but I don't think we want to be encouraging customizations around Prometheus just yet.  There is not any reasonable way for the customer to go about it.  I would probably remove most of this section

Comment (soulchips) [reply jekyll/_cci2/server-3-operator-metrics-and-monitoring.adoc:27]: Great catch. I think this should probably be specific to Telegraf customizations

Comment (soulchips) [reply jekyll/_cci2/server-3-operator-metrics-and-monitoring.adoc]: done

**Review by lloydChris — APPROVED** [review]
Nice work

**Comment by MikeyYeahYeah** [comment]
So much red <3

**Review by MikeyYeahYeah — APPROVED** [review]

**Review by annabaker — APPROVED** [review]

**Review by MikeyYeahYeah — APPROVED** [review]

**Pull request merged**